integreat_cms/cms/templates/analytics/__page_hix_row.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n %}
+{% load rules %}
+{% load content_filters %}
+{% load page_filters %}
+{% with page_translation.page.id as page_id %}
+    {% with page_translation.hix_score as hix %}
+        <tr class="{% if hix < hix_threshold %}bg-red-300{% endif %}">
+            <td>
+                <a href="{% url 'edit_page' region_slug=request.region.slug language_slug="de" page_id=page_id %}">
+                    {{ page_translation.title }}
+                </a>
+            </td>
+            <td>
+                {% if hix %}
+                    {{ hix }}
```

Comment:
I think only one decimal is enough, as we also only show one decimal in the page form (if I'm correct)

Reply:
```suggestion
                    {{ hix|floatformat:2 }}
```
Maybe two decimals?